ui/components/popover/popover.scss

```diff
@@ -17,9 +17,36 @@ button.popoverButton {
   }
 }
 
+$popover-zIndex: 100;
+
 div.popoverContainer {
   display: none;
   position: relative;
+  .popoverBackdrop {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100vw;
+    height: 100vh;
+    background: #00000040;
+    z-index: $popover-zIndex - 1;
+    cursor: pointer;
+    button {
+      /**
+       * Unfortunately, the Roll20/jquery setup fails if the button is styled as the popoverBackdrop.
+       * This needs to be a button with a parent, otherwise, we see errors like:
+       * ```
+       * vtt.bundle.js:formatted:556749 Uncaught TypeError: Cannot read properties of null (reading 'dataset')
+       *   at HTMLButtonElement.<anonymous> (vtt.bundle.js:formatted:556749:45)
+       *   at HTMLBodyElement.dispatch (jquery-1.9.1.js:3074:9)
+       *   at elemData.handle (jquery-1.9.1.js:2750:28)
```

Comment:
Popover backdrops are a bit tricky in Roll20.
My original attempt was to do `<button class="popoverBackdrop">Click me</button>`
However, this fails in Roll20
```
Uncaught TypeError: Cannot read properties of null (reading 'dataset')
    at HTMLButtonElement.<anonymous> (vtt.bundle.js:formatted:556749:45)
    at HTMLBodyElement.dispatch (jquery-1.9.1.js:3074:9)
    at elemData.handle (jquery-1.9.1.js:2750:28)
```
And looking at the source code:
The error occurs here:
`else if ($(this)[0].offsetParent.dataset.reprowid) {`
which fails due to `offsetParent` being undefined.

Buttons in general, appear to need a parent.

Instead, to work around this, I add a parent div, with `position: fixed`, which solves this issue.